_docs/setup/install-kubernetes.md

```diff
@@ -14,34 +14,36 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
+The following instructions assume you have access to a Kubernetes **1.7 or newer** cluster
+and  [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled.
+Always check the Release Notes of the Istio version you are installing for detailed requirements and limitations.
 
-* The following instructions assume you have access to a Kubernetes cluster with version > 1.7.0
-  and  [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled. To install Kubernetes locally,
-  try [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+* Depending on your Kubernetes provider:
+  * To install Istio locally, setup [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
 
-* If you are using [Google Container Engine](https://cloud.google.com/container-engine), find out your cluster
+  * If you are using [Google Container Engine](https://cloud.google.com/container-engine), find out your cluster
   name and zone, and fetch credentials for kubectl:
-  
-  ```bash
-  gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
-  ```
 
-* If you are using [IBM Bluemix Container Service](https://www.ibm.com/cloud-computing/bluemix/containers), find out your cluster name, and fetch credentials for kubectl:
+    ```bash
+    gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
+    ```
 
-  ```bash
-  $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
-  ```
-* If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
+  * If you are using [IBM Bluemix Container Service](https://www.ibm.com/cloud-computing/bluemix/containers), find out your cluster name, and fetch credentials for kubectl:
 
-  ```bash
- oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
- oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
-  ```  
+    ```bash
+    $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
+    ```
+  * If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
+
+    ```bash
+    oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
+    oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
+    ```  
 
 * Install the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or upgrade to the latest
-  version supported by your cluster.
+  version supported by your cluster (version 1.7 or newer for CRD support, and it should match your cluster's).
```

Comment:
Sorry, just caught this.. Bit colloquial. Better to say
Install or upgrade the kubernetes client [kubectl] and ensure that its of the same version as the targeted server.
or some nicer variant of this..

Reply:
k but I think lin wanted "CRD" part somewhere - I'll try something and ptal after next commit

Reply:
sure.. just nix the `it should match your cluster's ` and replace it with some thing more complete. The rest is fine.

Reply:
going with : 

* Install or upgrade the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to match
  version supported by your cluster (version 1.7 or newer for CRD support, and it should match your cluster's).